pool.go

```diff
@@ -59,6 +59,7 @@ retry:
 		}
 	}
 	p.cond.L.Unlock()
+	v.StopTimer()
```

Comment:
If the timer is not stopped successfully, we need to acquire another connection.

Reply:
Ah, that's right. Thanks!

Reply:
Fixed [390e19b](https://github.com/redis/rueidis/pull/727/commits/390e19b48220265d50a59d22624e8280b55b009b)